### ISSUE TITLE
Authorization error on Mac OS

### ISSUE THREAD

**Issue by eptaylor** [castlelemongrab/parlance#49]
Literally my first issue, so may be a user issue.

Running Mac OS 11.1, Node v. 14.15.3, npm 6.14.9, parlance v. 1.1210.1.  Getting a similar error as in Issue #22.

When I installed Parlance, it installed to /usr/local/lib/node_modules/@castlemongrab/parlance.  It did not create a config folder, but I created one manually.  Also manually created a cookies folder, into which I put the jst and mst files.

Doing "parlance init -o" with the various jst/mst files seems to work ("New file has been written to disk"), but when I "parlance profile" I get:

[fatal] Unable to read authorization data from config/auth.json

Any thoughts on what I'm doing wrong?

**Comment by inh2102** [comment]
create empty config folder in parlance

then 

parlance init —mst <mst-value> —jst <jst-value> -o config/auth.json

mst and jst values only, not cookie ‘files’

**Comment by inh2102** [comment]
@eptaylor 

I code in R and followed these instructions:

# Clone 'parlance' by castlelemongrab:
system('git clone http://github.com/castlelemongrab/parlance')

# Credentials...
do_credentials <- function(jst,mst) {
  system(
    paste0('cd parlance;','parlance init --mst ,',mst,
           ' --jst ',jst,' -o config/auth.json'))
}

# Follow inh2102/scrape_parler-R Github instructions for where to find mst and jst...
jst <- 'insert_here'
mst <- 'insert_here'
do_credentials(jst,mst)

**Comment by eptaylor** [comment]
Okay, I worked past that error.  Thanks for the R script.

Now, new error.  Similarly to issue #41, regardless of the command I issue into Parlance, after a 2 second pause, I get: 

[network] Fetching v1/profile
(node:18809) UnhandledPromiseRejectionWarning: StatusError: Unauthorized
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/@castlelemongrab/parlance/node_modules/bent/src/nodejs.js:133:23)
    at Object.onceWrapper (events.js:422:26)
    at ClientRequest.emit (events.js:315:20)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641:27)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)
    at TLSSocket.socketOnData (_http_client.js:509:22)
    at TLSSocket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at TLSSocket.Readable.push (internal/streams/readable.js:223:10)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:18809) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:18809) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.


I've tried these following commands and get the same error each time:
parlance followers seanhannity
parlance followers andyhogue (Note: a friend of mine who has a very small profile)


Thoughts on that one?  For whatever it's worth, I did go back in and confirm that my jst was up-to-date from a current session.
